packages/next-server/server/config.ts

```diff
@@ -17,6 +17,9 @@ const defaultConfig: { [key: string]: any } = {
   pageExtensions: ['tsx', 'ts', 'jsx', 'js'],
   target: process.env.__NEXT_BUILDER_EXPERIMENTAL_TARGET || 'server',
   poweredByHeader: true,
+  crossOrigin: 'anonymous',
+  modern: true,
```

Comment:
These options should be under experimental first

Reply:
Moved them inside `experimental`